great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+<!-- Main Cointainer-->
+<div id="main_container">
+  <!--Navigation Container-->
+  <div id="nav">
+    <nav>
+      <a href="/html/">Services</a> 
+      <a href="/css/">Product</a> 
+      <a href="/js/">Vision</a> 
+      <a href="/jquery/">Features</a>
+      <a href="/html/">About</a> 
+      <a href="/css/">Contact</a>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </nav>
+  </div>
+  <!--Heading Container-->
+  <div id="heading">
```

Comment:
I like the way you are thinking throughout the index in terms of separation of content and meaning! Let's try to take it a step further by using more of those semantic HTML elements that we learned about that gives more meaning to the content contained within, like `header`, `section`, etc